### ISSUE TITLE
[Bug] App crashes in arch linux.

### ISSUE THREAD

**Issue by bambirombi** [HACKERALERT/Picocrypt#75]
When dropping any type of file to picocrypt on arch linux it suddenly closes.

**Comment by bambirombi** [comment]
(Picocrypt:8121): dbind-WARNING **: 21:12:14.644: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-jedp9uwAl1: No such file or directory

**Comment by HACKERALERT** [comment]
Is your Arch using X11 or Wayland?

**Comment by HACKERALERT** [comment]
Also, try install dbus or the equivalent package.

**Comment by HACKERALERT** [comment]
If none of those fix it, try `export NO_AT_BRIDGE=1` before running Picocrypt.

**Comment by bambirombi** [comment]
I had dbus package so i tried export NO_AT_BRIDGE=1. Now it works correctly. Thanks!!!

**Comment by HACKERALERT** [comment]
For future reference: https://unix.stackexchange.com/a/230442/471231
